src/shared/varlink-io.systemd.Network.c

```diff
@@ -2,58 +2,613 @@
+                SD_VARLINK_FIELD_COMMENT("Routing protocol identifier"),
+                SD_VARLINK_DEFINE_FIELD(Protocol, SD_VARLINK_INT, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Human-readable routing protocol name"),
+                SD_VARLINK_DEFINE_FIELD(ProtocolString, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Whether this is a blackhole next hop (drops packets)"),
+                SD_VARLINK_DEFINE_FIELD(Blackhole, SD_VARLINK_BOOL, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Multipath group members with weights for load balancing"),
+                SD_VARLINK_DEFINE_FIELD_BY_TYPE(Group, Group, SD_VARLINK_ARRAY|SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Configuration source for this next hop"),
+                SD_VARLINK_DEFINE_FIELD(ConfigSource, SD_VARLINK_STRING, 0),
+                SD_VARLINK_FIELD_COMMENT("Address of the configuration provider"),
+                SD_VARLINK_DEFINE_FIELD(ConfigProvider, SD_VARLINK_INT, SD_VARLINK_ARRAY|SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Configuration state of this next hop"),
+                SD_VARLINK_DEFINE_FIELD(ConfigState, SD_VARLINK_STRING, SD_VARLINK_NULLABLE));
+
+static SD_VARLINK_DEFINE_STRUCT_TYPE(
+                LLDPNeighbor,
+                SD_VARLINK_FIELD_COMMENT("Chassis identifier in human-readable format"),
+                SD_VARLINK_DEFINE_FIELD(ChassisID, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Raw chassis identifier bytes"),
+                SD_VARLINK_DEFINE_FIELD(RawChassisID, SD_VARLINK_INT, SD_VARLINK_ARRAY),
+                SD_VARLINK_FIELD_COMMENT("Port identifier in human-readable format"),
+                SD_VARLINK_DEFINE_FIELD(PortID, SD_VARLINK_STRING, 0),
+                SD_VARLINK_FIELD_COMMENT("Raw port identifier bytes"),
+                SD_VARLINK_DEFINE_FIELD(RawPortID, SD_VARLINK_INT, SD_VARLINK_ARRAY),
+                SD_VARLINK_FIELD_COMMENT("Description of the port"),
+                SD_VARLINK_DEFINE_FIELD(PortDescription, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("System name of the LLDP neighbor"),
+                SD_VARLINK_DEFINE_FIELD(SystemName, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("System description of the LLDP neighbor"),
+                SD_VARLINK_DEFINE_FIELD(SystemDescription, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("Enabled capabilities bitmap (bridge, router, WLAN, etc.)"),
+                SD_VARLINK_DEFINE_FIELD(EnabledCapabilities, SD_VARLINK_INT, 0),
+                SD_VARLINK_FIELD_COMMENT("Manufacturer Usage Description URL (RFC8520)"),
+                SD_VARLINK_DEFINE_FIELD(MUDURL, SD_VARLINK_STRING, SD_VARLINK_NULLABLE),
+                SD_VARLINK_FIELD_COMMENT("VLAN identifier for this port"),
```

Comment:
"VLAN identifier for this port" doesn't make sense to me. Just "VLAN identified".